proto/kvrpcpb.proto

```diff
@@ -815,6 +814,9 @@ message ScanDetailV2 {
 
     // Total number of bytes from block reads.
     uint64 rocksdb_block_read_byte = 7;
+
+    // Total number of bytes read from RocksDB
```

Comment:
Considering our latest spec, maybe we should change the comment?

Reply:
Of course, I will change it before dev :)